### PR TITLE
Added uranus rings

### PR DESCRIPTION
**Pull request by Katniss218**
Uranus rings (config part of the PR)

### CODE REVIEW

File: GameData/RealSolarSystem/RSSKopernicus/Uranus/Uranus.cfg [hunk 1]
```diff
@@ -125,6 +125,22 @@
 				}
 			}
 		}
+		
+		Rings
+		{
+			Ring
+			{
+				angle = 74.8989043025419
```
Comment (lpgagnon) [inline GameData/RealSolarSystem/RSSKopernicus/Uranus/Uranus.cfg]: At least in my principia install, this angle puts the rings out of alignment with all the moons, as well as with the planet's rotation angle. angle=0 gives the expected result.

**Comment by siimav** [comment]
Needs https://github.com/KSP-RO/RSS-Textures/pull/45 to work properly

File: GameData/RealSolarSystem/RSSKopernicus/Uranus/Uranus.cfg (hunk 1 above)
Comment (siimav) [reply GameData/RealSolarSystem/RSSKopernicus/Uranus/Uranus.cfg]: If that's the case then you need to add another patch with `NEEDS[Principia]` and the different angle value

**Comment by NathanKell** [comment]
@lpgagnon do you have an angle that works well for Principia? Should be easy enough to do something like

angle:NEEDS[!Principia] = 74.8989043025419
angle:NEEDS[Principia] = (other angle)

**Comment by al2me6** [comment]
> @ lpgagnon do you have an angle that works well for Principia? Should be easy enough to do something like

Quoting https://github.com/KSP-RO/RealSolarSystem/pull/220#discussion_r541980236,

> angle=0 gives the expected result

**Comment by lpgagnon** [comment]
What I _don't_ know is if the current config is in fact valid for the non-principia case. If yes, then yes the pair of NEEDS would make sense

**Comment by siimav** [comment]
Welp, looks like you completely screwed up this PR now :(

**Comment by siimav** [comment]
Closing this because getting notifications about commit messages titled "*yes*" is mildly annoying.

**Pull request closed without merging**